docs/src/lateral/local-inertial.md

```diff
@@ -45,7 +45,7 @@ river_routing = "local-inertial" # default is kinematic-wave
 inertial_flow_alpha = 0.5      # alpha coefficient for model stability (default = 0.7)
 froude_limit = true            # default is true, limit flow to subcritical-critical according to Froude number
 h_thresh = 0.1                 # water depth [m] threshold for calculating flow between cells (default = 1e-03)
-riverlength_bc = 100000.0      # river length [m] for boundary points
+riverlength_bc = 1000.0        # river length [m] for boundary points (default = 1e04)
```

Comment:
This change looks good. Small detail: In the text you mention 10 km, but here it says 1 km. Also in the comments of src/flw.jl#L511 it says 1 km.

Reply:
Thanks for catching the typo in flow.jl comments, fixed this now in flow.jl. For the docs: a `riverlength_bc` of a 1000.0 m is specified in the TOML, while the default is 10 km (1e04 m as comment in line 48).